src/quo/components/drawers/drawer_top/view.cljs

```diff
@@ -121,6 +135,12 @@
      {:community-logo community-logo
       :community-name community-name}]
 
+    (= :account-context-tag type)
```

Comment:
<img width="291" alt="Screenshot 2024-01-16 at 20 55 55" src="https://github.com/status-im/status-mobile/assets/22799766/39e3c5b5-74d4-48da-bc20-7a5b39d88c03">
should this just be `:context-tag` to align with Figma 🤔

Reply:
Yes, It should be `:context-tag` with type `:account`, but we already have `(= :account type)` prop for `account-subtitle` https://github.com/status-im/status-mobile/blob/8f12c717ce4384806469ec903f4ccf5fb26e8e9b/src/quo/components/drawers/drawer_top/view.cljs#L120
and `(= :context-tag type)` for `context-tag-subtitle` with `:type :community`  https://github.com/status-im/status-mobile/blob/8f12c717ce4384806469ec903f4ccf5fb26e8e9b/src/quo/components/drawers/drawer_top/view.cljs#L133

That's why I introduced new mapping `:account-context-tag` -> `:context-tag` with type `:account`
... so many types....

Reply:
As alternative I can extend `(= :context-tag type)` with new prop for `context-tag` type

Reply:
Sure whatever suits best, it's mostly the outer api that we want to match to figma, so whatever reflects that best is good :)

Reply:
There is no separate design for [drawer-top](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=2500-42270&mode=design&t=SAwQatXp4Z0zsCzw-0)